Functions/TestDrive.Tests.ps1

```diff
@@ -171,32 +171,61 @@ InModuleScope Pester {
 # # this works correctly but needs administrator privileges to create the symlinks
 # # so I comment it out till I decide what to do with it, running always as admin just
 # # to test this is imho not a good idea. Tested on powershell 2 and 5 -- nohwnd
-# InModuleScope Pester {
-#     Describe "Clear-TestDrive" {
+InModuleScope Pester {
+
+    Describe "Clear-TestDrive" {
+        BeforeAll {
+            $SkipTest = $true
+            # test on non-windows PowerShell Core
+            if ( $PSVersionTable.PSVersion -ge "6.0" -and $IsWindows -eq $false ) {
```

Comment:
Try using `GetPesterPsVersion` and `GetPesterOs` to avoid fails. It works properly on Powershell 2. 
```
$SkipTest = (GetPesterOs) -ne "Windows" -and (GetPesterPsVersion) -ge 6
```

Reply:
thanks!